packages/nc-gui/components/smartsheet/header/Cell.vue

```diff
@@ -90,18 +90,27 @@ const onClick = (e: Event) => {
         'self-start': isForm || isSurveyForm,
       }"
     />
-    <div
+    <NcTooltip
       v-if="column"
-      class="name pl-1"
       :class="{
         'cursor-pointer pt-0.25': !isForm && isUIAllowed('fieldEdit') && !hideMenu && !isExpandedForm,
         'cursor-default': isForm || !isUIAllowed('fieldEdit') || hideMenu,
         '!truncate': !isForm,
       }"
-      :data-test-id="column.title"
+      class="name pl-1"
+      placement="bottom"
     >
-      {{ column.title }}
-    </div>
+      <template #title> {{ column.title }} </template>
+
+      <div
+        :class="{
+          '!truncate': !isForm,
```

Comment:
1. `!` not required.
2. since you change to `NcTooltip in L93, the truncate in L98 is no longer required.

Reply:
Got it. I think `!` is not required for both place tho. Please double check.

Reply:
1.) **fixed in below Commit**

[fix : removed unwanted ! - important](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)
[b862466](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)

Reply:
! removed in all places